packages/@aws-cdk/aws-ecr-assets/lib/adopted-repository.ts

```diff
@@ -22,6 +22,8 @@ interface AdoptedRepositoryProps {
  * adopting will make the repository "owned" by the stack. It will be cleaned
  * up when the stack gets deleted, to avoid leaving orphaned repositories on
  * stack cleanup.
+ *
+ * @deprecated this class is no longer used for ECR assets
```

Comment:
You can delete it. This file is not exported from `index.ts`.

Reply:
YASS!